msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -5,42 +5,47 @@ description: Identify the meeting apps API references with examples and Code sam
 ms.topic: conceptual
 ms.author: lajanuar
 ms.localizationpriority: medium
-keywords: teams apps meetings user participant role api usercontext notification signal query 
+keywords: teams apps meetings user participant role api user context notification signal query 
 ---
 
 # Meeting apps API references
 
-The meeting extensibilities provide APIs to transform the meeting experience:
+The meeting extensibility provide APIs to enhance meeting experience. You can perform the following with help of the listed APIs:
 
 * Build apps or integrate existing apps within meeting lifecycle.
-* Use the APIs to make your app aware of the meeting.
-* Select the APIs you want to use to enhance the meeting experience.
+* Use APIs to make your app aware of meeting.
+* Select required APIs to improve the meeting experience.
 
-The following table provides a list of APIs:
+The following table provides a list of APIs available across the Microsoft Teams Client (MSTC) and Microsoft Bot Framework (MSBF) SDKs:
 
-|API|Description|Request|Source|
-|---|---|----|---|
-|**GetUserContext**| Enables you to get contextual information to display relevant content in a Teams tab. |_**microsoftTeams.getContext( ( ) => {  /*...*/ } )**_|Microsoft Teams client SDK|
-|**GetParticipant**| Enables a bot to fetch participant information by meeting ID and participant ID. |**GET** _**/v1/meetings/{meetingId}/participants/{participantId}?tenantId={tenantId}**_ |Microsoft Bot Framework SDK|
-|**NotificationSignal** | Enables you to provide meeting signals that are delivered using the existing conversation notification API for user-bot chat. It allows you to signal based on user action that shows an in-meeting dialog box. |**POST** _**/v3/conversations/{conversationId}/activities**_|Microsoft Bot Framework SDK|
-|**Meeting Details** | Enables you to get static meeting metadata. |**GET** _**/v1/meetings/{meetingId}**_| Bot SDK |
-|**CART**|Enables you to post captions for a meeting, which was started.|**POST /cartcaption?meetingid=04751eac-30e6-47d9-9c3f-0b4ebe8e30d9&token=04751eac&lang=en-us HTTP/1.1**|Microsoft Teams client SDK|
+|Method| Description| Source|
+|---|---|----|
+|[**Get user context**](#get-user-context-api)| Get contextual information to display relevant content in a Teams tab.| MSTC SDK|
+|[**Get participant**](#get-participant-api)| Fetch participant information by meeting ID and participant ID. |MSBF SDK|
+|[**Send notification signal**](#send-notification-signal-api)| Provide meeting signals using the existing conversation notification API for user-bot chat and allows to notify user action that shows an in-meeting dialog box. |MSBF SDK|
+|[**Get meeting details**](#get-meeting-details-api)| Get a meeting's static metadata. |Bot SDK |
+|[**Send CART API**](#send-cart-api)|Enable you to post captions for a meeting, which was started. |MSTC SDK|
```

Comment:
Please maintain consistency with the rest of the values in this column. 

This could be called 'Send real-time captions'